sites/public/styles/overrides.scss

```diff
@@ -350,9 +350,12 @@ a.button.is-primary:hover {
 
 .hero__button__first {
   margin-bottom: 1rem;
+  background-color: white;
+  margin-right: 20px;
+  width: 20%;
```

Comment:
I think we probably want to set an actual width here. This is what it currently looks like on mobile:
![image](https://user-images.githubusercontent.com/371177/146064317-e8468ddd-d32e-41d5-828b-6066469985bf.png)

It looks like we also want to stack them on mobile:
https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497

Reply:
fixed